jacorb-name-server/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM jboss/base:latest
+MAINTAINER Gytis Trikleris <gytis@redhat.com>
+WORKDIR /opt/jboss
+EXPOSE 3528
+
+ENV JACORB_VERSION=2.3.1 \
+    JACORB_HOME=/opt/jboss/jacorb \
+    NAME_SERVER_PORT=3528 \
+    PATH=$PATH:/opt/jboss/jacorb/bin
+
+USER root
+
+RUN yum install -y ant \
+    && yum clean all
+
+USER jboss
+
+RUN curl -LOk http://www.jacorb.org/releases/$JACORB_VERSION/jacorb-$JACORB_VERSION-bin.zip \
```

Comment:
Well, other project have a single version and all Dockerfiles in the repo base on this version. Here we have 3 versions: WildFly, Jacorb and Narayana. Which one should be used is not clear.